ci/etc/quickstart-config.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(#3604) - figure out how to run pass arguments safely
```

Comment:
FYI, #3604 has been closed already. Not sure if there's a different issue to link to, or if this doesn't need one.

Reply:
Fixed.